### ISSUE TITLE
Linear gameplay mode

### ISSUE THREAD

**Issue by RubenTheDuck** [RF-Stuff/Bugs-Suggestions#65]
Fight your way through the map with your squad from one checkpoint to another

**Comment by Modelmat** [comment]
from @bramkaandorp
```
It would be cool to have the ability for certain flags to be locked (permanently owned by one team), or only capturable if one or more other specified flags are captured.

It would make a predetermined capturing order possible.

Forgotten Hope 2 had this perfected, and I'd love to play maps which use this type of gameplay.

It would also allow for maps with Conquest: Assault Lines type gameplay like in BF2142.
```

**Comment by BramKaandorp** [comment]
Adding to my earlier comment, here's a scenario to explain more specifically what I mean:

Let's say Eagle starts at flag 1, and they need to capture flag 4 to win (presuming that the game is set up with that end goal).

One way to set it up is to make Eagle capture flag 2 before flag 3 gets unlocked, after which flag 4 gets unlocked.

Another way to do it it is to leave the order of capture for flags 2 and 3 open to the player, as long as both are captured.

In short, map makers should also be able to give the player more freedom in their map, if they don't want to go for complete linearity or complete freedom.
